<launch>
  <node name="cable_point_cloud_removal" pkg="motoman_cable_removal" type="cable_point_cloud_removal_node" output="screen">
    <param name="loop_rate" value="10" />
    <param name="cable_start_pos_x" value="0.36" />
    <param name="cable_start_pos_y" value="0.00" />
    <param name="cable_start_pos_z" value="1.56" />

    <param name="source_pc_topic_name" value="/merged_cloud" />
    <param name="filtered_pc_topic_name" value="/cable_removed_pointcloud" />
    <param name="box_name" value="/remove_area" />
  </node>
</launch>
